nubis/puppet/consul.pp

```diff
@@ -12,6 +12,16 @@
   }
 }
 
+# initv startup script is borked
+case $::osfamily {
+  'redhat': {
+    package { "daemonize":
+      ensure => 'present',
+      install_options => [ '--enablerepo=epel' ],
+    }
+  }
```

Comment:
Good to have at least a `default` with a `notice` statement but this should work